Makefile

```diff
@@ -1,6 +1,9 @@
 .PHONY: release
 release:
-	dune build --profile=release @install
+	dune build -p varstored-guard @install
```

Comment:
This does not have the `release` profile. Does this risk a performance problem?

Reply:
`-p` implies `--profile release`, Dune wouldn't let me use both of them:
```
dune build -p varstored-guard --profile=release
dune: Cannot use --profile and -p simultaneously
```

Reply:
```
 -p PACKAGES, --for-release-of-packages=PACKAGES
           Shorthand for --root . --only-packages PACKAGE
           --ignore-promoted-rules --no-config --profile release.
```

Reply:
strange, it allows me it use both on dune 2.4.0